dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -219,7 +219,7 @@ class CDefaultContextMenu :
         STDMETHOD(GetSite)(REFIID riid, void **ppvSite) override;
 
         // IServiceProvider
-        virtual HRESULT STDMETHODCALLTYPE QueryService(REFGUID svc, REFIID riid, void**ppv)
+        virtual HRESULT STDMETHODCALLTYPE QueryService(REFGUID svc, REFIID riid, void**ppv) override
```

Comment:
From the definition of `STDMETHOD` for C++,
`#define STDMETHOD(method)        virtual HRESULT STDMETHODCALLTYPE method`
you could also take the opportunity to simplify the line:
```suggestion
        STDMETHOD(QueryService)(REFGUID svc, REFIID riid, void**ppv) override
```